SCSM.AzureAutomation/SCSM.AzureAutomation/Enumeration/RunbookStatus.mpx

```diff
@@ -3,11 +3,24 @@
     <EntityTypes>
       <EnumerationTypes>
         <EnumerationValue ID="AzureAutomationRunbook.Status" Accessibility="Public" />
-        <EnumerationValue ID="AzureAutomationRunbook.Status.Invalid" Parent="AzureAutomationRunbook.Status" Accessibility="Public" />
-        <EnumerationValue ID="AzureAutomationRunbook.Status.Active" Parent="AzureAutomationRunbook.Status" Accessibility="Public" />
-        <EnumerationValue ID="AzureAutomationRunbook.Status.Missing" Parent="AzureAutomationRunbook.Status" Accessibility="Public" />
-        <EnumerationValue ID="AzureAutomationRunbook.Status.PendingDelete" Parent="AzureAutomationRunbook.Status" Accessibility="Public" />
```

Comment:
I think we want to use the Runbook status to show it's status in SCSM 
like this in the Orchestrator Connector

![image](https://cloud.githubusercontent.com/assets/12474044/13549382/74128506-e2c9-11e5-89ad-cac296362fb4.png)